### PR TITLE
Export `zoo` method for `model_parameters` generic

### PR DESCRIPTION
**Pull request by jimrothstein**
Part of #746

### CODE REVIEW

File: R/1_model_parameters.R [hunk 1]
```diff
@@ -692,3 +692,8 @@ model_parameters.glm <- function(model,
   attr(out, "object_name") <- insight::safe_deparse_symbol(substitute(model))
   out
 }
+NULL
+#'	@title model_parameters.zoo
+#'	@description jr added Oct 17
+#'	@export
```
Comment (IndrajeetPatil) [inline R/1_model_parameters.R:698]:
```suggestion
#' @export
```

Comment (IndrajeetPatil) [reply R/1_model_parameters.R:698]: Other directives are not needed here.

**Review by IndrajeetPatil — CHANGES_REQUESTED** [review]
@jimrothstein Thanks!

This looks good, overall. Can you rebase with the main branch and see why there are changes to `.Rd` files? That is not expected.

Feel free to make these changes here, or open a new PR.

File: R/1_model_parameters.R (hunk 1 above)
Comment (jimrothstein) [reply R/1_model_parameters.R:698]: Will do ...  late here so fix tomorrow;  need to check .Rd

**Comment by jimrothstein** [comment]
Got the latest version (git pull upstream main) and then added TWO LINES:
`#' @export
model_parameters.zoo model_parameters.default`

Look any better?

**Comment by IndrajeetPatil** [comment]
Yes, but there is still the issue of changes to the `.Rd` files. Can you re-document and see what happens?

**Comment by jimrothstein** [comment]
On my local machine, I get 28 warnings when I document().   Need to install all the easystats packages, **I think**.  And `model_parameters.zoo` does NOT get documented.

Sorry, not as easy as I first thought.

**Comment by jimrothstein** [comment]
Have all packages .... document() .... no warnings.   Also ?model_parameters.zoo has no help.  Not sure about *.Rd files

File: R/1_model_parameters.R [hunk 2]
```diff
@@ -692,3 +692,5 @@ model_parameters.glm <- function(model,
   attr(out, "object_name") <- insight::safe_deparse_symbol(substitute(model))
   out
 }
```
Comment (IndrajeetPatil) [inline R/1_model_parameters.R:694]:
```suggestion
}

```

**Comment by IndrajeetPatil** [comment]
Looks good now. I will merge this once the builds pass.

**Review by IndrajeetPatil — APPROVED** [review]
Thanks, @jimrothstein! :)

Feel free to work more extensively on #746 in a new PR. You know the deal now. ;)

**Comment by jimrothstein** [comment]
Will do !   Later today.

**Pull request merged**